src/View/WorkController.ts

```diff
@@ -63,7 +63,7 @@ export class WorkController {
     )
   }
 
-  async upsertTimestamps(poetTimestamps: ReadonlyArray<PoetTimestamp>) {
+  async upsertTimestamps(poetTimestamps: ReadonlyArray<PoetBlockAnchor>) {
```

Comment:
Change `upsertTimestamps` to `upsertAnchors`
Change `poetTimestamps` to `poetAnchors`